### ISSUE TITLE
Location links not working

### ISSUE THREAD

**Issue by bsaiken** [FWAJL/FieldWorkManager#1280]
@gsouvik 
Unfortunately, I found a bug right away that will need to be fixed before we can address the Fine Uploader Milestone.  The Location Info window is not appearing when I click the Location in either the Location list or the Map (/mobile/ UI).

Also, please modify the CSS for the Location list so that the symbol appears on the same line as the Location Name.

![img_3447](https://cloud.githubusercontent.com/assets/8228624/9687037/b408b076-52ec-11e5-9372-626bb8e1e59b.PNG)

**Comment by bsaiken** [comment]
@gsouvik 
Jeremie may be working on this to address Issue #1270.

**Comment by gsouvik** [comment]
Brian, I have some emergency today and I'm away from my computer. I shall
try to be available tomorrow so that I may look into it.
Sincerely,
Souvik.

**Comment by bsaiken** [comment]
@gsouvik 
I think this is related to #1270, which Jeremie is working on.  I will need to put the Fine Uploader Issues on hold.  You can continue working on other Issues.

**Comment by FWAJL** [comment]
@bsaiken @gsouvik 

This has nothing to do with what I'm trying to do for #1270. I'm coding on a debug branch and release on a debug site, completely detached from the feature and dev branches and the dev site.

I can see nothing changed in the released commits in regard to this. And on the debug branch the popups work... I'm investigating.

**Comment by FWAJL** [comment]
@gsouvik @bsaiken 

This is the problem, unrelated to the work ongoing for #1270 
![screen shot 2015-09-07 at 06 50 49](https://cloud.githubusercontent.com/assets/7524388/9709500/e10d8528-552c-11e5-82db-71dac6d277ce.png)

I will try to find out what is missing.

**Comment by FWAJL** [comment]
@bsaiken do you want me to do that?

**Comment by bsaiken** [comment]
@FWAJL 
When the code is changed to use a new URL rather than a popup to display the mobile forms, this will need to be fixed, otherwise it will be impossible to conduct a function review.  Therefore, I am just going to close this Issue.
